examples/quality_classifier_example.py

```diff
@@ -59,7 +62,6 @@ def attach_args(
 ):
     argumentHelper = ArgumentHelper(parser)
 
-    argumentHelper.add_arg_device()
     argumentHelper.add_arg_enable_spilling()
```

Comment:
I think we should probably switch the whole thing to just just below , helps with overall alignment of NeMo-Curator. 

```python3
 args = ArgumentHelper.parse_distributed_classifier_args().parse_args()
 ```